js/apps/admin-ui/src/realm-settings/ThemesTab.tsx

```diff
@@ -50,178 +42,69 @@ export const RealmSettingsThemesTab = ({
         className="pf-v5-u-mt-lg"
         onSubmit={handleSubmit(save)}
       >
-        <FormGroup
-          label={t("loginTheme")}
-          fieldId="kc-login-theme"
-          labelIcon={
-            <HelpItem
-              helpText={t("loginThemeHelp")}
-              fieldLabelId="loginTheme"
-            />
-          }
-        >
-          <Controller
+        <FormProvider {...form}>
+          <DefaultSwitchControl
```

Comment:
This should probably come with some help text explaining this enables dark mode based on user preference when enabled, and does not force all themes into dark mode. It should also be mentioned that only themes that support dark mode will have it enabled.

Reply:
no only themes that use patternfly would be able to disable darkmode, when you implement it yourself this switch will have no effect

Reply:
even if your custom theme implements darkmode switching it off here will have no effect as it will only add `pf-v5-theme-dark` so unless you named your style the same this will work

Reply:
Yes, but a user might not understand this when enabling or disabling this toggle.